package.json

```diff
@@ -38,7 +38,8 @@
     "react-dom": "18.2.0",
     "react-hook-form": "^7.44.3",
     "react-icons": "^4.9.0",
-    "react-query": "^3.39.3"
+    "react-query": "^3.39.3",
+    "react-router-dom": "^6.14.1"
```

Comment:
We dont need `react-router-dom` as Next.js has its own file-system based routing built-in.
The `react-query` should be `@tanstack/react-query` instead as the new version of `react-query` has been renamed to `@tanstack/react-query`.